src/SCRIPTS/BF/ui.lua

```diff
@@ -133,38 +133,22 @@ local function processMspReply(cmd,rx_buf)
     end
 end
 
-local function MaxLines()
-    return #(Page.fields)
+local function incMax(val,inc,base)
```

Comment:
Using space after comma (like in natural language) makes it a bit easier to read.

Reply:
and here are the spaces ;-)